src/views/components/calendar/CalendarFooter.tsx

```diff
@@ -23,6 +24,12 @@ export default function CalendarFooter(): JSX.Element {
                 <Link className='linkanimate' href='https://github.com/Longhorn-Developers'>
                     <GithubIcon className='h-6 w-6' />
                 </Link>
+                <Link
+                    className='linkanimate'
+                    href='https://www.linkedin.com/company/longhorn-developers/posts/?feedView=all'
+                >
+                    <LinkedinIcon className='h-6 w-6' />
```

Comment:
Thanks for your PR! The LinkedIn icon is a bit smaller than its container, so it makes there be an inconsistent gap between the icons. If you force the x margin to be a bit smaller, it makes a more consistent gap between the icons

```suggestion
                    <LinkedinIcon className='h-6 w-6 -mx-0.75' />
```